app/models/user.rb

```diff
@@ -0,0 +1,39 @@
+require 'bcrypt'
+
+class User
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :name, String
+  property :email, String
+  property :username, String
+  property :password_digest, Text
+
+  has n, :peeps, :through => Resource
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)